src/players/VAST.js

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react'
+import vast from 'vast-client'
+import { callPlayer } from '../utils'
+import createSinglePlayer from '../singlePlayer'
+import { FilePlayer } from './FilePlayer'
+
+const MATCH_URL = /^(https?:\/\/)?((bs\.serving-sys\.com)|(pubads\.g\.doubleclick\.net))/
+export class VAST extends Component {
+  static displayName = 'VAST';
+  static canPlay = url => MATCH_URL.test(url);
+
+  state = {
+    sources: [],
+    tracker: null
+  }
+
+  callPlayer = callPlayer;
+
+  createSourceFiles (mediaFiles = []) {
+    return mediaFiles.reduce((arr, {fileURL: src, mimeType: type}) => {
```

Comment:
Should use a map instead since you are _always_ returning an array with the same element structure:
```javascript
createSourceFiles = (mediaFiles = []) => (
  mediaFiles.map(({ fileUrl: src, mimeType: type }) => { src, type })
);
```